Makefile

```diff
@@ -0,0 +1,19 @@
+IMAGE ?= prashcr/vim8s
+PLUGINS = test/plugins
+DOCKER = docker run -a stderr --rm -v $(PWD):/testplugin -v $(PWD)/test:/home "$(IMAGE)"
+
+$(PLUGINS)/vader.vim:
+	mkdir -p $(PLUGINS)
+	cd $(PLUGINS) && git clone https://github.com/junegunn/vader.vim.git
+
+test-setup: $(PLUGINS)/vader.vim
+	docker images -q $(IMAGE) || docker pull $(IMAGE)
+
+test: test-setup
+	vims=$$(docker run --rm $(IMAGE) ls /vim-build/bin | grep -E '^n?vim'); \
+	if [ -z "$$vims" ]; then echo "No Vims found!"; exit 1; fi; \
+	for vim in $$vims; do \
+	  $(DOCKER) $$vim '+Vader! test/*'; \
+	done
+
+.PHONY: test-setup test
```

Comment:
Makefile doesn't create files anymore. We could still add a make clean for docker containers but that would undermine caching and people could just do that manually if they happened to be running Docker on their own machine. And we don't need clean for Travis anyway.